api_app/api/routes/costs.py

```diff
@@ -0,0 +1,37 @@
+from datetime import date, timedelta
+from fastapi import APIRouter, Depends, Query
+
+from resources import strings
+from api.dependencies.database import get_repository
+from api.dependencies.workspaces import get_workspace_by_id_from_path
+from services.authentication import get_current_admin_user, get_current_workspace_owner_user
+from db.repositories.workspaces import WorkspaceRepository
+from db.repositories.workspace_services import WorkspaceServiceRepository
+from db.repositories.shared_services import SharedServiceRepository
+from models.domain.costs import CostReport, GranularityEnum, WorkspaceCostReport, generate_cost_report_stub, generate_workspace_cost_report_stub
+
+
+costs_core_router = APIRouter(dependencies=[Depends(get_current_admin_user)])
+costs_workspace_router = APIRouter(dependencies=[Depends(get_current_workspace_owner_user)])
```

Comment:
add admin to be able to pull workspace cost

Reply:
Hi @marrobi, @LizaShak raises here something that we need to decide upon:
cost report for WS level should be available both for Admin and WS owners,
as we see it there are two options:

1. Duplicate ```/api/workspaces/{workspace_id}/cost endpoint``` and have it also inside the ```cost_core_router``` (main /api)
2. Add admin user to each and every WS app with WS Owner role

What do we prefer?

Reply:
We have decided to implement option 1 as not all customers adds Admin user to WS Owner app registration.

Reply:
Do we have where else an endpoint that both the Workspace Owner and TRE Admin able to execute?

Reply:
Yes, currently three endpoints are set on ```workspaces_shared_router```:
[https://github.com/microsoft/AzureTRE/blob/main/api_app/api/routes/workspaces.py](https://github.com/microsoft/AzureTRE/blob/main/api_app/api/routes/workspaces.py)